add introduction

bg20876 opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

bg20876 committed:
add introduction

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hello, my name is Bill.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

bg20876 committed:
add additional content

add additional content as suggested in the comments section of the pull
request

bg20876 committed:
add deadly skills

adding more content

bg20876 committed:
add another deadly skill

Killing history using git rebase

codeschool-kiddo committed:
Added new rule about coffee

bg20876 committed:
Update dojo_rules.md

minor edits

bg20876 committed:
Merge pull request #2 from bg20876/new_rules

Too many people are spilling coffee

bg20876 committed:
Merge branch 'deadly_skills'

bg20876 committed:
Update Kill List

swap names for bad practices

bg20876 committed:
Merge branch 'release_branch_1.0'

bg20876 committed:
Add another programmer grievance

Added another personal programmer grievance to the list.  This fixes #3.

bg20876 committed:
Merge pull request #4 from bg20876/kill_list

Add another programmer grievance

bg20876 committed:
Update README.md

bg20876 committed:
Update README.md
